.travis.yml

```diff
@@ -9,10 +10,11 @@ addons:
     - ubuntu-toolchain-r-test
     packages:
     - g++-5
+    - clang
 
 install:
 - export CXX="g++-5"
-- script/configure
+- scan-build script/configure
```

Comment:
This might only be necessary if we were using autoconf:

> Running `configure` typically generates makefiles that have hardwired paths to the compiler, and by running `configure` through `scan-build` that path is set to `ccc-analyzer`.

Although `gyp` does insert a default value for `CC` and `CXX` into the makefile, you can still override it when running `make`, at least on my system:

```sh
$ script/configure
$ CXX=non-existent-compiler make

make: non-existent-compiler: No such file or directory
make: *** Error 1
```

So I'd think we could just run `script/test -b`. Does this work for you?